library/rtsp/src/main/java/com/google/android/exoplayer2/source/rtsp/RtspMediaSource.java

```diff
@@ -70,6 +71,7 @@
     private String userAgent;
     private boolean forceUseRtpTcp;
     private boolean debugLoggingEnabled;
+    private SocketFactory socketFactory;
```

Comment:
please move this field to after userAgent so that all object typed fields are grouped together.